app/assets/stylesheets/_globals.sass

```diff
@@ -19,69 +19,109 @@ header
       margin: 15px auto 0
       display: block
 
+.active-item
+  border: 3px solid $feed-item-border-color
+  color: $application-text-color
+  opacity: 1
+  .options
+    .option
+      label
+        color: $option-color
+        &:hover
+          background-color: $option-background-color-hover
+          color: $option-color-hover
+          cursor: pointer
+
+      input[type="radio"]:checked
+        & + label
+          background-color: $option-background-color-checked
+          box-shadow: inset 0 1px 6px $option-inset-shadow-color-checked
+          color: $option-color-checked
+
 .app
   @include outer-container
   margin-top: $logo-height * 2
-.feed
-  @include span-columns(8)
-  @include shift(2)
-  .item
-    background-color: $feed-item-background-color
-    border: 3px solid $feed-item-border-color
-    border-radius: 4px
-    min-height: $feed-item-height
-    padding: $feed-item-padding-vertical $feed-item-padding-horizontal
-    margin: $feed-item-margin
-
-.options
-  padding: 0
-
-  .option
-    list-style-type: none
-    display: inline
-
-    input[type="radio"]
-      display: none
-
-    label
-      background-color: $option-background-color
-      border: 1px solid $option-border-color
-      color: $option-color
-      padding: 5px 15px
-      width: 3rem
-      &:hover
-        background-color: $option-background-color-hover
-        color: $option-color-hover
-        cursor: pointer
-
-    input[type="radio"]:checked + label
-      background-color: $option-background-color-checked
-      box-shadow: inset 0 1px 6px $option-inset-shadow-color-checked
-      color: $option-color-checked
-
-  .option:first-of-type label
-    border-left: 1px solid $option-border-color
-    border-bottom-left-radius: 4px
-    border-top-left-radius: 4px
-
-  .option:last-of-type label
-    border-bottom-right-radius: 4px
-    border-top-right-radius: 4px
-
-  .option + .option label
-    border-left: none
+  .feed
+    margin-top: $logo-height
+    @include span-columns(8)
+    @include shift(2)
+    overflow: hidden
+    margin-bottom: 45vh
```

Comment:
Op mijn telefoon is dit nog wat te veel; wellicht ook even een mobile variant toevoegen?
![photo on 25-04-16 at 12 55](https://cloud.githubusercontent.com/assets/8157262/14781798/0c2ec03c-0ae5-11e6-8fc0-4bb5f386b698.jpg)